src/fire.js

```diff
@@ -10,8 +10,9 @@ const config = {
 };
 
 const db = firebase.initializeApp(config);
+const facebookProvider = new firebase.auth.FacebookAuthProvider()
 
 const dashboardsRef = db.database().ref();
 export const dashesRef = dashboardsRef.child('dashboards');
 
-export default db;
```

Comment:
What is the reason to change export type of a function and, subsequently change every instance this function is imported in all the components maintained by other developers?